test/Makefile

```diff
@@ -0,0 +1,68 @@
+.PHONY: default pr all
+
+default: check-deps kernel-config
+pr: check-deps kernel-config ../artifacts/test.img.tar.gz
+all: check-deps kernel-config ltp ../artifacts/test.img.tar.gz ../artifacts/test-ltp.img.tar.gz
+
+MOBY:=$(shell command -v moby 2> /dev/null)
+LINUXKIT:=$(shell command -v linuxkit 2> /dev/null)
+
+.PHONY: check-deps
+check-deps:
+ifndef MOBY
+	$(error "moby is not available. please install it.")
+endif
+ifndef LINUXKIT
+	$(error "linuxkit binary not found. please install it.")
+endif
+
+define check_test_log
+	@cat $1 |grep -q 'test suite PASSED'
+endef
+
+../artifacts:
+	mkdir -p $@
+
+test-kernel-config-initrd.img: $(MOBY) cases/test-kernel-config.yml
+	$(MOBY) build --pull cases/test-kernel-config.yml
+
+# uses qemu
+.PHONY: kernel-config
+kernel-config: $(LINUXKIT) test-kernel-config-initrd.img test-kernel-config-kernel test-kernel-config-cmdline
+	$(LINUXKIT) run qemu test-kernel-config 2>&1 | tee test.log
+	$(call check_test_log, test.log)
+
+# qemu only at this time
+.PHONY: qemu-efi
+efi: $(LINUXKIT) test-kernel-config-efi.iso
```

Comment:
Yes just re-using it to test EFI boot, although this isn't being done in CI just yet.